sbom/cnquery_sbom.proto

```diff
@@ -254,4 +257,13 @@ message Evidence {
   // the asset. The format and interpretation of this value depend on the
   // 'type'. For example, it could be a file path for file evidence.
   string value = 2;
-}
\ No newline at end of file
+}
+
+message Kernel {
```

Comment:
please remember message annotations as well :)

Reply:
thanks for adding it